### PR TITLE
Merge Story1 into Main - Story1 Trainer Review

### PR DESCRIPTION
**Pull request by vincedefreitas**
I believe I've met the requirements for story one. I just require trainer review and sign off of database structure. Looking at definition of done, I've done the following:

Followed dev process
Met story requirements
Responsive
Accessible (I think)
Unit Tested and Browser Tested

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+function getAllGames(object $db): array {
+    $query = $db->prepare('SELECT * FROM `videogames`;');
```
Comment (mallowmew) [inline src/functions.php]: We don't like `SELECT *` in production. While `SELECT *` is fine for discovering which fields are available, it has performance impacts from how it does that. Select the fields you want to display manually please.

File: tests/displayAllGamesTest.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+use PHPUnit\Framework\TestCase;
+
+class displayAllGamesTest extends TestCase {
+    public function testDisplayGamesMalformedInputs(): void
+    {
+        $games = "This should be an array";
+        $this->expectException(TypeError::class);
+        displayAllGames($games);
+    }
```
Comment (mallowmew) [inline tests/displayAllGamesTest.php]: Surely you can write a success test as well?

**Review by mallowmew — CHANGES_REQUESTED** [review]

File: src/functions.php (hunk 1 above)
Comment (vincedefreitas) [reply src/functions.php]: Done :) Please can you review?

File: tests/displayAllGamesTest.php (hunk 2 above)
Comment (vincedefreitas) [reply tests/displayAllGamesTest.php]: Done :) Please can you review?

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.card-img {
+    display: block;
+    width: 100%;
+    max-height: 200px;
+    /*aspect-ratio: 16 / 9;*/
+    object-fit: cover;
+    object-position: center;
+    /*border-top-right-radius: 0.25rem;*/
+    /*border-top-left-radius: 0.25rem;*/
```
Comment (mallowmew) [inline styles.css]: remove commented-out code - if you need it it's safely tucked away in the git history where you can find it again, don't need to keep commented stuff hanging around

File: tests/displayAllGamesTest.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class displayAllGamesTest extends TestCase {
+
+    public function testDisplayGamesOutput(): void
```
Comment (mallowmew) [inline tests/displayAllGamesTest.php]:
```suggestion
    public function testDisplayGamesOutput_Success(): void
```
Including if this is a success, failure or malformed test in the function name would be standard

**Review by mallowmew — CHANGES_REQUESTED** [review]

File: styles.css (hunk 3 above)
Comment (vincedefreitas) [reply styles.css]: Done :)

**Pull request merged**